velox/vector/BiasVector.h

```diff
@@ -147,7 +147,13 @@ class BiasVector : public SimpleVector<T> {
   }
 
   uint64_t retainedSize() const override {
-    return BaseVector::retainedSize() + values_->capacity();
+    uint64_t totalStringBufferSize{0};
```

Comment:
We no longer need to override this API, do we? We can also remove 'virtual' from this API in the BaseVector.